HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -138,6 +174,17 @@ void HLTScoutingEgammaProducer::produce(edm::StreamID sid, edm::Event& iEvent, e
     return;
   }
```

Comment:
The behaviour implemented here is that - if any of these input collections is missing - an empty output is produced and no errors are given.

Is it the intended behaviour ?
It seems it would make debugging quite hard if the output is empty and and one does not understand why.

If **it is** the intended behaviour, I would suggest to clean up the implementation: for example, declaring all Handles up front and having a single `if` block, or a `do {}` block with many `if` conditions.

No changes needed for this PR, but it could be cleaned up in a later PR.